lib/engine/game/g_18_zoo/game.rb

```diff
@@ -1273,15 +1327,22 @@ def update_zoo_tickets_value(turn, round_num = 1)
           end
         end
 
-        def bonus_after_par(corporation, money, _additional_tracks)
+        def bonus_after_par(corporation, money, additional_tracks, available_tracks)
           bank.spend(money, corporation)
           @log << "#{corporation.name} earns #{format_currency(money)} as treasury bonus"
-          # TODO: enable again after adding G18ZOO::Step::AdditionalTracksAfterPar
-          # @round.additional_tracks = additional_tracks
+
+          @round.available_tracks.concat(available_tracks)
+
+          @round.bonus_tracks = additional_tracks
         end
 
         def event_new_train!
-          # @new_train_brought = true # TODO: enable again after adding G18ZOO::Step::BuyTrain
+          @round.new_train_brought = true
+        end
+
+        def event_rust_own_3s_4s!
+          @log << '3S long and 4S owned by current player are rusted!'
```

Comment:
Something like this would be more consistent with other event log messages (the `--` will automatically make it bold):

```suggestion
          @log << '-- Event: 3S long and 4S owned by current player are rusted! --'
```

Reply:
done in 6e5282e3